app/models/fact.rb

```diff
@@ -4,6 +4,11 @@
 # Factable objects: Person
 class Fact < ApplicationRecord
   include ParseableDate
+  include SqlFunctions
+
+  module Types
+    BIRTH_DATE = 'birth'
```

Comment:
very minor: in general I'm trying to roughly conform to the [GEDCOM(X)](https://github.com/FamilySearch/gedcomx/blob/master/specifications/fact-types-specification.md) standard of naming Facts (and Events) so I think this would be better as `BIRTH`. (my bad for not documenting this!)

Reply:
Oh okay, I can switch it over

Reply:
That's done